### PR TITLE
Assignments/array functions

### PR DESCRIPTION
**Pull request by AWalnut1**
hehe funi pull request i have no idea what im doing

### CODE REVIEW

**Comment by AWalnut1** [comment]
i forgot to assign you as a reviewer

**Comment by AWalnut1** [comment]
speaking of requesting you as a reviewer i just finished it ahhahaahah

File: assignments/array-functions/code.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      return true
+  }
+  return false
+};
+
+const pyramid = (number) => {
+  let theMid = []
+  for (let i = 0; i < number; i++) {
+    for (let j = -1; j < i; j++) {
+      console.log(j)
```
Comment (gigamonkey) [inline assignments/array-functions/code.js:41]: Strictly speaking functions should not have extra side effects like this. So you should delete lines that you put in for debugging once you're done.

**Review by gigamonkey — APPROVED** [review]
Looks good. 👍🏻 You can go ahead and merge this.

File: assignments/array-functions/code.js (hunk 1 above)
Comment (AWalnut1) [reply assignments/array-functions/code.js:41]: fair

**Pull request merged**